libs/langchain/pyproject.toml

```diff
@@ -140,6 +140,7 @@ rspace_client = {version = "^2.5.0", optional = true}
 upstash-redis = {version = "^0.15.0", optional = true}
 google-cloud-documentai = {version = "^2.20.1", optional = true}
 fireworks-ai = {version = "^0.6.0", optional = true, python = ">=3.9,<4.0"}
+astrapy = {version = ">=0.5.3", optional = true}
```

Comment:
We actually don't include these here anymore - should just raise the proper exception from the integration!